### ISSUE TITLE
Image elements do not have [alt] attributes + Links do not have a discernible name

### ISSUE THREAD

**Issue by adamkubina1** [adamkubina1/SWAPni-TO#57]
User Avatars missing alt and <a/> needs a discernible name -> fix inside UserAvatar component.
